tests/hook_main.py

```diff
@@ -835,6 +836,73 @@ def ticket_delete(transaction):
 
 
 
+# ---------------------- Attendees (Ticket Holder) ---------------------
+
+@hooks.before("Attendees > Attendees Collection > List All Attendees")
```

Comment:
fix this everywhere in the doc i.e remove the line between `-----....` and `@hooks.befo....` everywhere